src/ui/components/HourChecker/HourChecker.tsx

```diff
@@ -136,7 +138,11 @@ const HourChecker = memo((props: HourCheckerProps) => {
     );
 });
 
-const useStyles = makeStyles<{ width: string }>({ "name": { HourChecker } })((theme, { width }) => ({
+const useStyles = makeStylesEdt<{ width: string }>({ "name": { HourChecker } })((theme, { width }) => ({
```

Comment:
Could (re)explain me with sometimes you use const useStyles for CSS and sometimes the sx attribute ? ?

Reply:
It does the same thing. Sometimes the syntax of useStyles is heavy for just one prop (an import, and 2 more const). For example, when we only need to set the prop display: "none" or "visible" deppending on a bool, we find the useStyle syntax heavier so we use sx. When there is more than 1 prop to edit at once, we use useStyle. 
I fixed all other encountered cases (sx with more than 1 prop) by using useStyle.